metrics/web/lib/dashboard/presentation/widgets/project_groups_dropdown_body.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:selection_menu/components_configurations.dart';
+
+/// A widget that displays the dropdown body.
+class ProjectGroupsDropdownBody extends StatefulWidget {
```

Comment:
It seems to not have any Metrics dependencies - should it be a base widget? As we have similar dropdowns in Users screen - maybe it can be reused there?